src/sentry/static/sentry/app/components/avatarCropper.tsx

```diff
@@ -0,0 +1,470 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {AVATAR_URL_MAP} from 'app/constants';
+import {t, tct} from 'app/locale';
+import Well from 'app/components/well';
+import {AvatarUser} from 'app/types';
+
+const resizerPositions = {
+  nw: ['top', 'left'],
+  ne: ['top', 'right'],
+  se: ['bottom', 'right'],
+  sw: ['bottom', 'left'],
+};
+
+type Position = keyof typeof resizerPositions;
+
+type Props = {
+  model: AvatarUser;
+  updateDataUrlState: (opts: {savedDataUrl?: string | null; dataUrl?: string}) => void;
+  type: 'user' | 'team' | 'organization' | 'project';
+  savedDataUrl: string;
+};
+
+type State = {
+  file: File | null;
+  objectURL: string | null;
+  mousePosition: {pageX: null | number; pageY: null | number};
```

Comment:
@EvanPurkhiser Seems weird to me to have `null` before `number` like this

Reply:
yeah just noticed this